plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -25,16 +25,27 @@
 
 
 HAZARDS_MAPPING = {
-    "cae": "Child abuse and exploitation",
+    "cae": "Child sexual exploitation",
     "cbr": "Chemical, biological, radioloigcal, nuclear and high-yield explosives weapons",
-    "gra": "Graphic depictions of hazards",
-    "ssh": "Suicide, self-harm and eating disorders",
+    "hat": "Hate",
+    "nvc": "Non-violent crimes",
+    "sc": "Sex-related crimes",
+    "ssh": "Suicide & self-harm",
     "ter": "Terrorism",
+    "vio": "Violent crimes",
 }
+_PERSONA_TYPES = ["typical", "malicious", "vulnerable"]
+
+
+class SafeTestItemContext(BaseModel):
+    persona_type: str
 
 
 class SafeTestResult(BaseModel):
-    frac_safe: float
+    """Fraction safe and number of measured test items for each persona type."""
+
+    frac_safe: Mapping[str, float]
```

Comment:
These two maps have to contain the same keys in order to work. IMO that means we should represent it as a single map with a value combining the counts.

Maybe introduce a `PersonaResult` class?

Reply:
That's a good idea, done.